src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3175,6 +3183,59 @@ void MusicXMLParserDirection::dynamics()
     }
 }
 
+void MusicXMLParserDirection::otherDirection()
+{
+    // <other-direction> element is used to define any <direction> symbols not yet in the MusicXML format
+    const String smufl = m_e.attribute("smufl");
+    const Color color = Color::fromString(m_e.attribute("color"));
+
+    // Read smufl symbol
+    if (!smufl.empty()) {
+        SymId id = SymNames::symIdByName(smufl, SymId::noSym);
+        if (id != SymId::noSym) {
+            Symbol* smuflSym = Factory::createSymbol(m_score->dummy());
+            smuflSym->setSym(id);
+            if (color.isValid()) {
+                smuflSym->setColor(color);
+            }
+            m_elems.push_back(smuflSym);
+        }
+        m_e.skipCurrentElement();
+    } else {
+        // TODO: Multiple sets of maps for exporters other than Dolet 6/Sibelius
+        // TODO: Add more symbols from Sibelius
+        std::map<String, String> otherDirectionStrings;
+        if (m_pass1.exporterString().contains(u"Dolet")) {
+            otherDirectionStrings = {
+                { String(u"To Coda"), String(u"To Coda") },
+                { String(u"Segno"), String(u"<sym>segno</sym>") },
+                { String(u"CODA"), String(u"<sym>coda</sym>") },
+            };
+        }
+        std::map<String, SymId> otherDirectionSyms;
+        otherDirectionSyms = { { String(u"Rhythm dot"), SymId::augmentationDot },
```

Comment:
For now, maybe nicer to do this one like this:
```suggestion
        static const std::map<String, SymId> otherDirectionSyms = { { String(u"Rhythm dot"), SymId::augmentationDot },
```